exonum/src/sandbox/consensus/transactions.rs

```diff
@@ -232,6 +232,27 @@ fn duplicate_tx_in_pool() {
     ));
 }
 
+#[test]
+fn rebroadcast_transactions() {
+    let sandbox = timestamping_sandbox();
+
+    let mut transactions = TimestampingTxGenerator::new(DATA_SIZE)
+        .take(5)
+        .collect::<Vec<_>>();
+
+    transactions.sort_by(|a, b| a.hash().cmp(&b.hash()));
```

Comment:
Can we move this into separete sandbox_helper, i am sure we can replace sorting algorithm later.